blocks/form/form.js

```diff
@@ -159,6 +182,7 @@ async function createForm(formURL) {
   const formTag = document.createElement('form');
   const fields = data.map((fd) => renderField(fd));
   formTag.append(...fields);
+  await decorateForm(formTag);
```

Comment:
guard against the `decorateForm` not being present, find a way to import it lazily.

Reply:
Done